packages/core/src/codewhisperer/service/transformationHubViewProvider.ts

```diff
@@ -115,9 +115,10 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
 
     private async showPlanProgress(startTime: number): Promise<string> {
         const planProgress = getPlanProgress()
-        let planSteps = undefined
-        if (planProgress['generatePlan'] === StepProgress.Succeeded) {
+        let planSteps = transformByQState.getPlanSteps()
+        if (planProgress['generatePlan'] === StepProgress.Succeeded && transformByQState.isRunning()) {
             planSteps = await getTransformationSteps(transformByQState.getJobId())
+            transformByQState.setPlanSteps(planSteps)
```

Comment:
Store the plan steps so that if network error happens, the previously-retrieved steps remain visible in Transformation Hub.